### PR TITLE
QA approved build

### PR DESCRIPTION
**Pull request by hcheung01**
Changes:

-Added -k flag to curl command in Dockerfile when downloading TurboVNC from Scourgeforge because their SSL cert expired
-Couple of broken URIs for some commands in Jupyter Notebook
-Some commands was using the wrong compiler, ex: pgcc for fortran files
-Order of files to compile

### CODE REVIEW

File: labs/module4/English/Fortran/README.ipynb [hunk 1]
```diff
@@ -114,7 +114,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgfortran -fast -ta=multicore -Minfo=accel -o laplace_multicore laplace2d.f90 jacobi.f90 && ./laplace_multicore"
+    "!pgf90 -fast -ta=multicore -Minfo=accel -o laplace_multicore laplace2d.f90 jacobi.f90 && ./laplace_multicore"
```
Comment (jefflarkin) [inline labs/module4/English/Fortran/README.ipynb]: This should use `pgfortran`, `pgf90` is deprecated.

File: labs/module4/English/Fortran/README.ipynb [hunk 2]
```diff
@@ -301,7 +301,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgf90 -fast -ta=tesla:cc30 -Minfo=accel -o laplace_data_clauses laplace2d.f90 jacobi.f90  && ./laplace_data_clauses 1024 1024"
+    "!pgf90 -fast -ta=tesla -Minfo=accel -o laplace_data_clauses laplace2d.f90 jacobi.f90  && ./laplace_data_clauses 1024 1024"
```
Comment (jefflarkin) [inline labs/module4/English/Fortran/README.ipynb]: Should be `pgfortran`

File: labs/module4/English/Fortran/README.ipynb [hunk 3]
```diff
@@ -404,7 +404,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgcc -fast -ta=tesla:managed -Minfo=accel -o laplace_managed laplace2d.f90 jacobi.f90  && ./laplace_managed"
+    "!pgf90 -fast -ta=tesla:managed -Minfo=accel -o laplace_managed laplace2d.f90 jacobi.f90  && ./laplace_managed"
```
Comment (jefflarkin) [inline labs/module4/English/Fortran/README.ipynb]: Should be pgfortran.

File: labs/module4/English/C/README.ipynb [hunk 4]
```diff
@@ -300,7 +300,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgcc -fast -ta=tesla -Minfo=accel -o laplace_data_clauses jacobi.c laplace2d.c && ./laplace_data_clauses 1024 1024"
+    "!pgcc -fast -ta=tesla,managed -Minfo=accel -o laplace_data_clauses jacobi.c laplace2d.c && ./laplace_data_clauses 1024 1024"
```
Comment (jefflarkin) [inline labs/module4/English/C/README.ipynb]: I believe at this point in the tutorial we want the managed flag removed to necessitate the addition of data clauses.

File: labs/module4/English/Fortran/README.ipynb (hunk 1 above)
Comment (hcheung01) [reply labs/module4/English/Fortran/README.ipynb]: Fixed and create new pull request.

File: labs/module4/English/C/README.ipynb (hunk 4 above)
Comment (hcheung01) [reply labs/module4/English/C/README.ipynb]: removed managed flag. I sent the wrong build.

File: labs/module4/English/Fortran/README.ipynb (hunk 2 above)
Comment (hcheung01) [reply labs/module4/English/Fortran/README.ipynb]: Fixed and create new pull request.

File: labs/module4/English/Fortran/README.ipynb (hunk 3 above)
Comment (hcheung01) [reply labs/module4/English/Fortran/README.ipynb]: Fixed and create new pull request.

**Pull request merged**